plugins/scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts

```diff
@@ -37,7 +37,7 @@ import {
   TaskSpecV1beta3,
   TaskStep,
 } from '@backstage/plugin-scaffolder-common';
-import { UserEntity } from '@backstage/catalog-model';
+import { createCounterMetric, createHistogramMetric } from '../../../../catalog-backend/src/util/metrics';
```

Comment:
I'm wondering if these need to be properly exported in the `catalog-backend` and then you can add that package and import them from there here

Reply:
Definitely, thx for the catch

Reply:
Do you think we should move that whole class to backend-common instead?

Reply:
Yeah, this gets into some territory that I'm not sure what the maintainers will want to do, they are on an offsite so we won't know till next week. If I was in your shoes I would just do the export for now as that shouldn't be too much work to undo if they don't want to go that route

Reply:
Same with this, looks good 👍